products/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+
+class MainCategory(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = 'main_categories'
+
+
+class SubCategory(models.Model):
+    name = models.CharField(max_length=100)
+    main = models.ForeignKey('MainCategory', on_delete=models.CASCADE)
```

Comment:
필드명은 최대한 명확하게 지어주세요.
```suggestion
    main_category = models.ForeignKey('MainCategory', on_delete=models.CASCADE)
```

Reply:
넵 디테일한 지적 감사합니다!